lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -207,7 +207,8 @@ def test_dashboard_analytics_points_at_insights(self):
         Test analytics dashboard message is shown
         """
         response = self.client.get(self.url)
-        analytics_section = '<li class="nav-item"><a href="" data-section="instructor_analytics">Analytics</a></li>'
+        button = '<li class="nav-item"><button type="button" class="btn-link" data-section="instructor_analytics">Analytics</button></li>'  # pylint: disable=line-too-long
```

Comment:
Nit: since you are disabling the line length check anyway, might as well get rid of the temp variable.